sdk/common/version/version.go

```diff
@@ -8,9 +8,9 @@ import (
 
 // values set by linker using ldflag -X.
 var (
-	ver    string // nolint:gochecknoglobals // set by linker
-	date   string // nolint:gochecknoglobals // set by linker
-	commit string // nolint:gochecknoglobals // set by linker
+	ver    string //nolint:gochecknoglobals // set by linker
```

Comment:
I think you can put the `//nolint` comment at the top of the block instead of each line:
```
//nolint:gochecknoglobals // set by linker
var (
    ver   string
    ...
)
```